apps/comps/types/competition.ts

```diff
@@ -68,6 +68,14 @@ export interface Competition {
       votedAt?: string;
     };
   };
+  portfolioValue?: number;
```

Comment:
Since these types being a bit disconnected from the actual api code base, I kept these all optional.